json_parse_node.go

```diff
@@ -185,7 +185,11 @@ func (n *JsonParseNode) GetObjectValue(ctor absser.ParsableFactory) (absser.Pars
 			field := fields[key]
 			if field == nil {
 				if value != nil && isHolder {
-					itemAdditionalData[key] = value.value
+					rawValue, err := value.GetRawValue()
```

Comment:
before defaulting to the raw value, we should probably try to go through all the scalar types and the collections of scalar types.